cmd/system-probe/modules/eventmonitor.go

```diff
@@ -70,6 +70,18 @@ func createEventMonitorModule(_ *sysconfigtypes.Config, wmeta optional.Option[wo
 		}
 		evm.RegisterEventConsumer(network)
 		log.Info("event monitoring network consumer initialized")
+
+		netconfig := netconfig.New()
+		if netconfig.EnableUSMEventStream && netconfig.ServiceMonitoringEnabled {
```

Comment:
Isn't this already checked here? https://github.com/datadog/datadog-agent/blob/main/cmd/system-probe/modules/eventmonitor_linux.go#L31